app/src/main/java/org/mozilla/focus/tabs/TabsChromeListener.java

```diff
@@ -22,6 +22,13 @@
  */
 public interface TabsChromeListener {
 
+    enum Factor {
```

Comment:
What about using IntDef instead of enum to avoid increasing to much size to the dex?